trustedApplications/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*   Profile Editing Pane
+**
+** Unlike most panes, this is available any place whatever the real subject,
+** and allows the user to edit their own profile.
+**
+** Usage: paneRegistry.register('profile/profilePane')
+** or standalone script adding onto existing mashlib.
+*/
+
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import solidUi, { SolidUi } from 'solid-ui'
+import { IndexedFormula } from 'rdflib'
+import paneRegistry from 'pane-registry'
+
+import { PaneDefinition } from '../types'
+import { Container } from './container'
+
+const nodeMode = (typeof module !== 'undefined')
+
+let panes
+let UI: SolidUi
+
+if (nodeMode) {
```

Comment:
That's not there for testing - it's primarily there because it already was there; see [here](https://github.com/solid/solid-panes/blob/f2b7c78a1b669f9ba235b0ace1b02a91e40b9f49/trustedApplications/trustedApplicationsPane.js#L13).

I think it may have to do with having been a Firefox extension or something? Perhaps Tim knows more about it.